client/ayon_motionbuilder/version.py

```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+"""Package declaring AYON addon 'motionbuilder' version."""
+__version__ = "0.1.0"
```

Comment:
It feels invalid that this is `0.1.0` and the `package.py` version is `0.1.0-dev.1` - shouldn't those always be in sync?